### ISSUE TITLE
PDF bookmarks are malformed

### ISSUE THREAD

**Issue by BryantLam** [openshmem-org/specification#66]
Document edits to the `\openshmem` LaTeX macro has broken the rendering of PDF bookmarks. Text "OpenSHMEM" is now blank. (E.g., "History of" instead of "History of OpenSHMEM")

**Comment by nspark** [comment]
Oops. That one is on me. I'll look into fixing it. Good catch!

**Comment by nspark** [comment]
It seems like defining `\openshmem` with a default argument causes issues with the hyperref package. For example:
```latex
\newcommand{\openshmem}{Open\-SHMEM} % works with hyperref
\newcommand{\openshmem}[1]{Open\-SHMEM~#1} % works with hyperref
\newcommand{\openshmem}[1][]{Open\-SHMEM~#1} % does not work with hyperref
```

Also, searching around returned results of others having issues with `\ifthenelse` and hyperref as well.

I'll keep digging, but maybe I'll have to drop the idea of supporting both styles of commands:
```latex
\openshmem      % --> OpenSHMEM
\openshmem[1.3] % --> OpenSHMEM 1.3
```

**Comment by nspark** [comment]
This issue would be closed by #91.
